picocrt/machine/x86/crt0-32.S

```diff
@@ -89,6 +92,14 @@ _start32:
 	addl	$12, %esp
 
 #ifdef PICOLIBC_TLS
```

Comment:
I bet there's a way to do this as part of the general memory setup by placing the GDT in the .data segment instead of the .rodata segment.